index.js

```diff
@@ -0,0 +1,7 @@
+// transpile:main
+
+
+import { default as authorize } from './lib/authorize.js';
```

Comment:
```js
import authorize from './lib/authorize';
```

1) when importing default you can just give it a name without brackets
2) you don't need the `.js` when importing